### ISSUE TITLE
ip address after reconected to redis

### ISSUE THREAD

**Issue by arteck** [ioBroker/ioBroker.js-controller#1598]
we need in log a ip address from redis master after reconnect.

**Comment by Apollon77** [comment]
We would need to find out how to get the current connected IP from ioredis ... found nothing in docs

**Comment by erichsch** [comment]
redis-cli -p 26379
sentinel get-master-addr-by-name <your cluster id>  # get current sentinel master IP

**Comment by Apollon77** [comment]
@erichsch Yes this is true, but we do not have an sentinel connection ourself ... all this is handled by the ioredis library  internally ... And I'm not sure if I want to open another connection just to add the IP to the log ...

**Comment by foxriver76** [comment]
> sentinel get-master-addr-by-name

Actually it is
```
127.0.0.1:26379> sentinel get-master-addr-by-name mymaster
1) "127.0.0.1"
2) "6379"
```
